src/app/analytics/analytics.factory.js

```diff
@@ -530,6 +530,7 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, sessionSer
               name: cartItem.designationNumber,
               id: cartItem.designationNumber,
               price: cartItem.amount.toString(),
+              processingFee: cartItem.amountWithFees ? Number.parseFloat(cartItem.amountWithFees - cartItem.amount).toFixed(2) : undefined,
```

Comment:
I'm trying to parse the value after subtraction, so I don't think so, unless we want 3 calls.

Reply:
I might be misunderstanding, but if `amountWithFees` and `amount` are both numbers, you can just do `(cartItem.amountWithFees - cartItem.amount).toFixed(2)` because JavaScript doesn't have separate types for ints and floats. `Number.parseFloat` isn't necessary unless you're dealing with strings.